apf-ingester/apf_Ingester.py

```diff
@@ -0,0 +1,264 @@
+import re
+import os
+import json
+import shutil
+import tempfile
+import pathlib
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+from typing import Dict, List, Optional
+
+
+def cleanup_source_document(sgm_path: str) -> None:

```

Comment:
style: Don't use strings for paths - use `pathlib.Path`.

Reply:
I actually need to delete this function, as it is not being used anymore. I will replace `str` with `Path` everywhere else